joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -73,6 +73,10 @@ trait AstCreatorHelper(implicit withSchemaValidation: ValidationMode) { this: As
       case IdentifierName                                 =>
         // TODO: Look at scope object for possible types
         Defines.Any
+      case PredefinedType =>
+        node.code match
```

Comment:
This should be more general
```suggestion
        BuiltinTypes.DotNetTypeMap(node.code).getOrElse(Defines.Any)
```